### ISSUE TITLE
pretrained model on flowers102

### ISSUE THREAD

**Issue by hahaha811** [SHI-Labs/Compact-Transformers#27]
Hi, Firstly, Thank for your great work.

I am trying to use pretrained model on flowers-102 and the accuracy I could achieve is around 8%. Could you provide some details for pretrained model on flowers102? Btw, other pretrained model is working in my code. My setting  for flowers102

initialization of net
```
elif 'cct_7' in arch:
    model = getattr(models, arch)(img_size=224,
                                    num_classes=num_classes,
                                    positional_embedding='sine',
                                    n_conv_layers=2,
                                    kernel_size=7) 

elif 'cct_14' in arch:
    model = getattr(models, arch)(img_size=384,
                                    num_classes=num_classes,
                                    positional_embedding='learnable',
                                    n_conv_layers=2,
                                    kernel_size=7)
```

dataloader
```
elif args.data == 'flowers102':
  traindir = os.path.join(args.data_root, 'flower_data/train')
  valdir = os.path.join(args.data_root, 'flower_data/valid')
  testdir = os.path.join(args.data_root, 'flower_data/test')
  normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                   std=[0.229, 0.224, 0.225])

  train_set = datasets.ImageFolder(traindir, transforms.Compose([
      transforms.RandomResizedCrop(224),
      transforms.RandomHorizontalFlip(),
      transforms.ToTensor(),
      normalize
  ]))
  
  val_set = datasets.ImageFolder(valdir, transforms.Compose([
      transforms.Resize(256),
      transforms.CenterCrop(224),
      transforms.ToTensor(),
      normalize
  ]))
```
Thank you

**Comment by alihassanijr** [comment]
Hi, thank you for your interest.

I've noticed a couple of issues. For starters, you're using ImageNet-1k statistics (mean and std). I'm also not sure about the script because I'm not seeing the full picture.

Either way, we're pushing some major changes to the repo today, and with it provide:
* Configuration files containing the exact training settings for each checkpoint we release
* A new and more complicated training script (from `timm`, that's how we've been training most of our models recently)
* Configuration files for datasets -- this is specifically going to be useful since we've already found good and useful settings for some datasets without needing to change anything in the `timm` training script.

Please stay tuned and try to train it later with the new changes. If you're still having issues after that, feel free to re-open the issue.
